autosklearn/experimental/askl2.py

```diff
@@ -51,8 +51,13 @@ def __call__(
         initial_configurations = []
         for member in self.portfolio.values():
             try:
+                _member = {
+                    key: member[key]
+                    for key in member
+                    if key in scenario.cs.get_hyperparameter_names()
```

Comment:
Minor optimization, can we compute `scenario.cs.get_hyperparameter_names()` before the dict comprehension? The function will get called for every `key` in `memeber`.

```python
def f():
    print("hello")
    return ["a", "b", "c"]

x = [key for key in "abcdefghi" if key in f()]
# hello
# hello
# hello
# ...
```